### PR TITLE
Auto markdown formatting and typo fixes

### PR DESCRIPTION
**Pull request by tas50**
Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: src/supermarket/docs/CONFIGURING.md [hunk 1]
```diff
@@ -1,60 +1,36 @@
...
+Supermarket uses [dotenv](https://github.com/bkeepers/dotenv) to handle setting environment variables. These are the different configurations needed to get Supermarket running. The default parameters provided in `.env` should be sufficient to run the test suite.
 
 ## Recommended System Specs
 
 If you are running Supermarket in a production environment, these are the minimum, basic specifications we recommend for your Supermarket server:
-* At least 1 CPU
-* At least 1 GB Memory
-* At least 5 GB free space in /var
+
+- At least 1 CPU
```
Comment (iennae) [inline src/supermarket/docs/CONFIGURING.md]: I know it's not new content, but this paragraph is weird. It's labelled "Recommended System Specs" and then "minimum, basic specification". If it's "Minimum system specs" then you could drop "At least" from each of these bullet points.

File: src/supermarket/docs/CONFIGURING.md [hunk 2]
```diff
@@ -68,125 +44,94 @@ Your Supermarket will use email to send notifications to users (i.e. when a cook
...
-* `no_crawl`, Adds noindex, nofollow meta tags so search engines won't crawl Supermarket.
-* `fieri`, Enables the ability for Cookbook Versions to be evaluated by Fieri
-* `announcement`, Enables the foldable annoucement banner that is displayed on all views.
-  when uploaded
-* `collaborator_groups`, Enables collaborator groups, which allow you to manage collaborators
-   through groups
+Certain features of Supermarket can be enabled or disabled using `FEATURES`. The `FEATURES` ENV variable should be defined as a comma separated list of features to be enabled. Supermarket supports the following features.
+
+- `cla`, Enables the ability to sign the ICLA and CCLA, as well as Curry
+- 'gravatar', Enables Supermarket to use gravatars for user avatar icons
```
Comment (iennae) [inline src/supermarket/docs/CONFIGURING.md]: again, know it's not new, but should this be `gravatar` rather than 'gravatar'?

**Comment by robbkidd** [comment]
@tas50 What tool made these markdown format changes automatically?

**Comment by tas50** [comment]
@robbkidd https://atom.io/packages/tidy-markdown

Just a warning: It destroys tables from time to time

**Comment by robbkidd** [comment]
Have we decided to was that formatter for markdown across projects?

**Comment by tas50** [comment]
@robbkidd The same way we decided on cookstyle ;)

**Comment by robbkidd** [comment]
Sure. However cookstyle is a CLI tool that can be added to CI for enforcement. Relying on an editor plugin will take us down the path of PRs with walls of red/green changes unrelated to actual content changes.

**Comment by robbkidd** [comment]
tenor-154018501

**Pull request merged**